### PR TITLE
issue/4 Added Adapt.a11y.ariaLevel

### LINKED ISSUE #4: Aria level calculation should be accessible to react
Currently the helper [`a11y_attrs_heading` ](https://github.com/adaptlearning/adapt-contrib-core/blob/master/js/helpers.js#L341) produces a string of attributes which isn't compatible with react.

### PR DESCRIPTION
**Pull request by oliverfoster**
fixes #4 

### Changed
* Moved aria level calculation from `a11y_attrs_heading` helper to `Adapt.a11y.ariaLevel`
* Fixed some linting issues

### CODE REVIEW

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

File: js/a11y.js [hunk 1]
```diff
@@ -151,6 +152,35 @@ class A11y extends Backbone.Controller {
...
+    let level = 1;
+
+    // first check to see if the Handlebars context has an override
+    if (this._ariaLevel) {
+      levelOrType = this._ariaLevel;
+    }
+
+    if (isNaN(levelOrType) === false) {
+      // if a number is passed just use this
+      level = levelOrType;
```
Comment (eleanor-heath) [inline js/a11y.js:173]: return early here

File: js/a11y.js [hunk 2]
```diff
@@ -151,6 +152,35 @@ class A11y extends Backbone.Controller {
...
+    } else if (_.isString(levelOrType)) {
```
Comment (eleanor-heath) [inline js/a11y.js:174]: this can be an `if` instead of `else if`

File: js/a11y.js [hunk 3]
```diff
@@ -151,6 +152,35 @@ class A11y extends Backbone.Controller {
...
+      // if a string is passed check if it is defined in global configuration
+      cfg._ariaLevels = cfg._ariaLevels || defaultAriaLevels;
+      if (cfg._ariaLevels?.['_' + levelOrType] !== undefined) {
```
Comment (eleanor-heath) [inline js/a11y.js:177]: return early here?

**Review by eleanor-heath — APPROVED** [review]
left a few suggestions around returning early but otherwise looks good.

File: js/helpers.js [hunk 4]
```diff
@@ -1,16 +1,5 @@
...
-  '_menuGroup': 2,
-  '_menuItem': 2,
-  '_page': 1,
-  '_article': 2,
-  '_block': 3,
-  '_component': 4,
-  '_componentItem': 5,
-  '_notify': 1
-};
+import a11y from './a11y';
```
Comment (eleanor-heath) [inline js/helpers.js:2]: should a11y be capitalised to A11y?

Comment (oliverfoster) [reply js/helpers.js:2]: no, it's an instance at this point, not a class.

**Review by chris-steele — APPROVED** [review]

**Pull request merged**